soccer/joystick/SpaceNavJoystick.cpp

```diff
@@ -34,50 +39,55 @@ void SpaceNavJoystick::update() {
         if (sev.type == SPNAV_EVENT_MOTION) {
             //  note: spacenav axes range from -350 to 350
 
-            //  spacenav x is side-to-side and z is forward-to-backward
-            //  strafe/translate
-            //  translation starts out as a normalized value from (-1,-1) to
-            //  (1,1)
-            _controlValues.translation =
-                Geometry2d::Point(sev.motion.x / 350.0, sev.motion.z / 350.0);
-
-            //  create a mouse deadzone - this keeps the robot from moving if
-            //  the joystick is shifted just ever so slightly
-            if (_controlValues.translation.mag() < 0.05) {
-                _controlValues.translation = Geometry2d::Point(0, 0);
-            }
-
-            //  twisting the spacenav corresponds to sev.motion.ry
-            _controlValues.rotation = sev.motion.ry / 350.0;
-            if (abs<float>(_controlValues.rotation) < 0.05) {
-                _controlValues.rotation = 0;
-            }
-
-            //  the y axis corresponds to pulling the knob up and down, which we
-            //  use to control the dribbler
-            //  moving the knob in other directions unintentionally moves the y
-            //  axis, so we give it a large deadzone
-            const int DribblerAxisDeadzone = 200;
-
             //  we don't want to increment it at each iteration, otherwise it
```

Comment:
Let's move this comment down a bit so it's right above the timing it describes: `(now - _lastDribbleTime)`.